packages/checkbox/src/mwc-checkbox-base.ts

```diff
@@ -167,32 +169,43 @@ export class CheckboxBase extends FormElement {
       </div>`;
   }
 
-  private _handleFocus() {
-    this.focused = true;
```

Comment:
Oh, looks like this is missing now. Can you add it back to `handleRippleFocus` and `handleRippleBlur`?

Reply:
Oops...